appium/api-mocks/apis/fes/fes-endpoints.ts

```diff
@@ -69,13 +69,28 @@ export const getMockFesEndpoints = (mockConfig: MockConfig, fesConfig: FesConfig
       // body is a mime-multipart string, we're doing a few smoke checks here without parsing it
       if (req.method === 'POST') {
         expectContains(body, '-----BEGIN PGP MESSAGE-----');
-        expectContains(body, '"associateReplyToken":"mock-fes-reply-token"');
-        expectContains(body, '"to":["to@example.com"]');
-        expectContains(body, '"cc":[]');
-        expectContains(body, '"bcc":["bcc@example.com"]');
-        authenticate(req);
-        expectContains(body, '"from":"user@disablefesaccesstoken.test:8001"');
-        return { url: `${mockConfig}/message/FES-MOCK-MESSAGE-ID` };
+        const match = String(body).match(/Content-Type: application\/json\s*\n\s*(\{.*\})/);
+
+        if (!match) {
+          throw new FesHttpErr('Bad request', Status.BAD_REQUEST);
+        }
+        const contentType = JSON.parse(match[0]);
+        const { associateReplyToken, to, cc, bcc } = contentType;
```

Comment:
It looks strange when `contentType` contains recipients and reply token, let's call it `messageData`

Reply:
Ok